pypsa/optimization/global_constraints.py

```diff
@@ -85,6 +86,11 @@ def define_tech_capacity_expansion_limit(n: Network, sns: Sequence) -> None:
             )
 
 
+@deprecated(
+    deprecated_in="1.0",
+    removed_in="1.1",
```

Comment:
We wanna follow semver now, which means everything what is in `1.0` needs to stay until `2.0`
```suggestion
    removed_in="2.0",
```